src/viewport/index.tsx

```diff
@@ -123,7 +126,11 @@ class Viewport extends React.Component<
         this.handleTimeChange = this.handleTimeChange.bind(this);
 
         this.visGeometry = new VisGeometry(loggerLevel);
+        this.disableCache = props.disableCache;
```

Comment:
I don't think you need to save this on the class here? you can just use the prop below

Reply:
yes, change made